### PR TITLE
Fix - various 0.0.7 issues

### PR DESCRIPTION
**Pull request by mormaer**
# Checklist
- [x] I have described what this PR contains

**Choose one of the following two options:**
- - [x] This PR does not introduce major changes
- - [ ] This PR introduces major changes, and I have consulted @buresdv, @jboi or @mormaer in the *Mlem Development Matrix room*

**Choose one of the following two options:**
- - [ ] This PR does not change the UI in any way
- - [x] This PR adds new UI elements / changes the UI, and I have attached pictures or videos of the new / changed UI elements

# Pull Request Information

## About this Pull Request
This PR address a couple of issues that have been discussed today in the development chat ahead of a planned release.

- Multiple error alerts showing while loading feeds
> This appears to be for SSL related errors on Beehaw where I can test, but some screenshots showed the _unhandled_ case firing, which is most likely a decoding related error. Until we can reproduce these and properly debug there cause I have silenced those alerts while the post feed is not empty as they are not _blocking_ problems.
- Ensure the body for certain post types are rendered as they were being hidden 🙈 
> Image and Web based posts were not rendering their bodies, ironically the post announcing this release didn't render the release notes in the app. See screenshots below.
- Added a TODO for further investigating duplicate posts, it's _much_ rarer but can still happen on occasion.

## Screenshots and Videos
| BEFORE | AFTER |
| --- | --- |
| Simulator Screenshot - iPhone 14 Pro - 2023-06-16 at 17 52 54 | Simulator Screenshot - iPhone 14 Pro - 2023-06-16 at 17 40 19 |

### CODE REVIEW

File: Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift [hunk 1]
```diff
@@ -478,6 +478,12 @@ struct CommunityView: View
                 account: account
             )
         } catch APIClientError.networking {
+            // TODO: we're seeing a number of SSL related errors on some instances while loading pages from the feed
+            // while we investigate the reasons we will only show this error if the user would otherwise be left with an empty feed
```
Comment (ShadowJonathan) [inline Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift:482]: Turn these into a bug issue, maybe?

Comment (mormaer) [reply Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift:482]: 👍 I'll raise some for the outstanding ones when I get back to a computer

Comment (buresdv) [reply Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift:482]: What do you guys think, safe to merge in the current state?

Comment (mormaer) [reply Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift:482]: I've added three issues for these:

#179 
#180 
#181 

The fixes included in this PR were put up as I thought we'd have wanted to address them prior to rolling the build out for public testing.

The frequent alerts will likely be annoying to users, and not rendering post bodies for web/image posts seems like a blocking issue.

If we're not planning to roll out a fix in the short term these could be left out to see if we reach a better resolution, however having them in place at least fixes the immediate problems from a UX perspective 🙏

Comment (mormaer) [reply Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift:491]: I've dropped one of the commits that was on this branch as it is now resolved by #193

**Review by ShadowJonathan — APPROVED** [review]
LGTM

**Review by buresdv — APPROVED** [review]

**Pull request merged**